site/en/docs/extensions/mv3/content_scripts/index.md

```diff
@@ -154,18 +159,21 @@ They can include JavaScript files, CSS files, or both. All auto-run content scri
   </tbody>
 </table>
 
+{% if false %}
```

Comment:
It wasn't here before, but can you add a quick comment as to why this is restricted?

Reply:
The extensions team is working on this feature, but it's not currently supported. At the moment our docs avoid mentioning features or capabilities that haven't at least landed in Canary.